hs_core/views/__init__.py

```diff
@@ -339,6 +339,31 @@ def delete_resource(request, shortkey, *args, **kwargs):
 
     return HttpResponseRedirect('/my-resources/')
 
+
+def rep_res_bag_to_irods_user_zone(request, shortkey, *args, **kwargs):
+    if request.is_ajax:
```

Comment:
Is this function ever be called other than via ajax call? If not then there is no need for this ```if``` statement. We can just document that this function needs to be called via ajax call.